examples/run_all.sh

```diff
@@ -6,6 +6,7 @@ set -e
 (cd gp/pseudovector && ./gpmsa_pseudovector gpmsa_input.txt)
 (cd bimodal && ./bimodal_gsl bimodal_1chain.inp)
 (cd gravity && ./gravity_gsl gravity_inv_fwd.inp)
+(cd line2D_with_Sensitivity && ./line2D_gsl slope_inv_fwd.inp)
```

Comment:
This may need to be extended to include running the pre-processing script the generates the input files needed by the application.